From bb45041bdc146e781cdf3c211d0b9e33768a2f25 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 21 May 2020 14:07:54 -0400
Subject: [PATCH] disable broken warning for auto var init

---
 build/config/compiler/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index e5314977a9dd7..a9e34d3cacdf0 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -348,7 +348,7 @@ config("compiler") {
     }
 
     if (is_clang && !is_nacl && !using_sanitizer) {
-      cflags += [ "-ftrivial-auto-var-init=zero", "-enable-trivial-auto-var-init-zero-knowing-it-will-be-removed-from-clang" ]
+      cflags += [ "-ftrivial-auto-var-init=zero", "-enable-trivial-auto-var-init-zero-knowing-it-will-be-removed-from-clang", "-Wno-unused-command-line-argument" ]
     }
 
     # Linker warnings.
-- 
2.39.1.windows.1

