From 300a29b78878b55c5d6be8bd20cec52e67d7ffe1 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 6 Mar 2018 00:27:41 -0500
Subject: [PATCH] disable showing popular sites by default

---
 components/ntp_tiles/features.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/components/ntp_tiles/features.cc b/components/ntp_tiles/features.cc
index d35052b2fec1c..07072abe6c695 100644
--- a/components/ntp_tiles/features.cc
+++ b/components/ntp_tiles/features.cc
@@ -14,7 +14,7 @@ const char kPopularSitesFieldTrialName[] = "NTPPopularSites";
 
 BASE_FEATURE(kPopularSitesBakedInContentFeature,
              "NTPPopularSitesBakedInContent",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
              "NTPMostLikelyFaviconsFromServer",
@@ -22,6 +22,6 @@ BASE_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
 
 BASE_FEATURE(kUsePopularSitesSuggestions,
              "UsePopularSitesSuggestions",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
-}  // namespace ntp_tiles
+}  // namespace ntp_tiles
\ No newline at end of file
-- 
2.35.1.windows.2

