From b8fa04b8125f952c564021d7ca15625584c13e41 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH] enable split cache by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 0f7b57a7fc718..b725434997231 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -87,7 +87,7 @@ BASE_FEATURE(kSplitCacheByNetworkIsolationKey,
 
 BASE_FEATURE(kSplitHostCacheByNetworkIsolationKey,
              "SplitHostCacheByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
              "PartitionConnectionsByNetworkIsolationKey",
-- 
2.35.1.windows.2

