From 684c072b3ab80706a985d7cfffcea57aaf32f384 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 8 Mar 2021 16:53:47 -0500
Subject: [PATCH] enable partitioning connections by default

---
 net/base/features.cc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 4d788e440c76a..daf06ee74b078 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -91,7 +91,7 @@ BASE_FEATURE(kSplitHostCacheByNetworkIsolationKey,
 
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
              "PartitionConnectionsByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kForceIsolationInfoFrameOriginToTopLevelFrame,
              "ForceIsolationInfoFrameOriginToTopLevelFrame",
@@ -99,15 +99,15 @@ BASE_FEATURE(kForceIsolationInfoFrameOriginToTopLevelFrame,
 
 BASE_FEATURE(kPartitionHttpServerPropertiesByNetworkIsolationKey,
              "PartitionHttpServerPropertiesByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPartitionSSLSessionsByNetworkIsolationKey,
              "PartitionSSLSessionsByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPartitionNelAndReportingByNetworkIsolationKey,
              "PartitionNelAndReportingByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kEnableDoubleKeyNetworkAnonymizationKey,
              "EnableDoubleKeyNetworkAnonymizationKey",
-- 
2.39.1.windows.1

