From 25d6369ca91a1d3a90f542f51d1a04e4af1feafb Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable Partitioned Cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index cbcd682c18943..d7acafcb26f23 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -275,7 +275,7 @@ BASE_FEATURE(kSamePartyAttributeEnabled,
 
 BASE_FEATURE(kPartitionedCookies,
              "PartitionedCookies",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNoncedPartitionedCookies,
              "NoncedPartitionedCookies",
-- 
2.39.1.windows.1

