From 503226ea76f0a6d5ab184d9dea428f4d96faaf9a Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable Partitioned Cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index bcea21e6a9ac1..d43d14df1b4a8 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -252,7 +252,7 @@ BASE_FEATURE(kSamePartyAttributeEnabled,
 
 BASE_FEATURE(kPartitionedCookies,
              "PartitionedCookies",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNoncedPartitionedCookies,
              "NoncedPartitionedCookies",
-- 
2.39.1.windows.1

