From aa304d710e286261640944ff04eee148577e78cb Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable Partitioned Cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 6cc44e981f558..d55e824e1be54 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -284,7 +284,7 @@ const base::Feature kSamePartyCookiesConsideredFirstParty{
     "SamePartyCookiesConsideredFirstParty", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kPartitionedCookies{"PartitionedCookies",
-                                        base::FEATURE_DISABLED_BY_DEFAULT};
+                                        base::FEATURE_ENABLED_BY_DEFAULT};
 const base::Feature kPartitionedCookiesBypassOriginTrial{
     "PartitionedCookiesBypassOriginTrial", base::FEATURE_DISABLED_BY_DEFAULT};
 
-- 
2.35.1.windows.2

