From e5d8d5635fe633d707aa5bf2cb45ae188ff8591c Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable Partitioned Cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 166b073e8e40b..8a4abfa35d2f7 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -282,7 +282,7 @@ const base::Feature kSamePartyCookiesConsideredFirstParty{
     "SamePartyCookiesConsideredFirstParty", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kPartitionedCookies{"PartitionedCookies",
-                                        base::FEATURE_DISABLED_BY_DEFAULT};
+                                        base::FEATURE_ENABLED_BY_DEFAULT};
 const base::Feature kPartitionedCookiesBypassOriginTrial{
     "PartitionedCookiesBypassOriginTrial", base::FEATURE_DISABLED_BY_DEFAULT};
 
-- 
2.35.1.windows.2

