From 599790e8856e49ef3c6ac95f129ca7e9bcb2a659 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable AppContainer by default

---
 sandbox/policy/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 368a01dfcd04d..7c3be4f8eff35 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -30,14 +30,14 @@ const base::Feature kWinSboxDisableExtensionPoints{
 
 // Enables GPU AppContainer sandbox on Windows.
 const base::Feature kGpuAppContainer{"GpuAppContainer",
-                                     base::FEATURE_DISABLED_BY_DEFAULT};
+                                     base::FEATURE_ENABLED_BY_DEFAULT};
 
 // Enables GPU Low Privilege AppContainer when combined with kGpuAppContainer.
 const base::Feature kGpuLPAC{"GpuLPAC", base::FEATURE_ENABLED_BY_DEFAULT};
 
 // Enables Renderer AppContainer
 const base::Feature kRendererAppContainer{"RendererAppContainer",
-                                          base::FEATURE_DISABLED_BY_DEFAULT};
+                                          base::FEATURE_ENABLED_BY_DEFAULT};
 
 // Enables shared/fixed policy for Windows sandbox policies.
 const base::Feature kSharedSandboxPolicies{"SharedSandboxPolicies",
-- 
2.35.1.windows.2

