From ca30f5fdaf7abe8641ef3d19318437166f84536c Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable AppContainer by default

---
 sandbox/policy/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 4dfbf9307ced7..9c8b879f9cc85 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -34,7 +34,7 @@ BASE_FEATURE(kWinSboxDisableExtensionPoints,
 // Enables GPU AppContainer sandbox on Windows.
 BASE_FEATURE(kGpuAppContainer,
              "GpuAppContainer",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables GPU Low Privilege AppContainer when combined with kGpuAppContainer.
 BASE_FEATURE(kGpuLPAC,
@@ -44,7 +44,7 @@ BASE_FEATURE(kGpuLPAC,
 // Enables Renderer AppContainer
 BASE_FEATURE(kRendererAppContainer,
              "RendererAppContainer",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables shared/fixed policy for Windows sandbox policies.
 BASE_FEATURE(kSharedSandboxPolicies,
-- 
2.35.1.windows.2

