From 0b1093d4add9b0ac0f9004d3c14d7027e67da9a6 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 11 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable AppContainer by default

---
 sandbox/policy/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 23b194d6c090b..5bb61d6714443 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -28,7 +28,7 @@ BASE_FEATURE(kWinSboxDisableExtensionPoints,
 // Enables GPU AppContainer sandbox on Windows.
 BASE_FEATURE(kGpuAppContainer,
              "GpuAppContainer",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables GPU Low Privilege AppContainer when combined with kGpuAppContainer.
 BASE_FEATURE(kGpuLPAC,
@@ -38,7 +38,7 @@ BASE_FEATURE(kGpuLPAC,
 // Enables Renderer AppContainer
 BASE_FEATURE(kRendererAppContainer,
              "RendererAppContainer",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables shared/fixed policy for Windows sandbox policies.
 BASE_FEATURE(kSharedSandboxPolicies,
-- 
2.35.1.windows.2

