From d4f7e62a07a4284e72769957afdc0d5dd0e2258d Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Wed, 28 Sep 2022 00:00:00 +0000
Subject: [PATCH] enable Network sandbox by default

---
 sandbox/policy/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 010281086e2ab..2901c7a1737f7 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -15,7 +15,7 @@ namespace sandbox::policy::features {
 // (Only causes an effect when feature kNetworkService is enabled.)
 BASE_FEATURE(kNetworkServiceSandbox,
              "NetworkServiceSandbox",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif  // !BUILDFLAG(IS_MAC) && !BUILDFLAG(IS_FUCHSIA)
 
 #if BUILDFLAG(IS_WIN)
-- 
2.39.1.windows.1

