From 10e27205a77787fcbbb200e8f3a795c2a0786e37 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Wed, 28 Sep 2022 00:00:00 +0000
Subject: [PATCH] disable extension points

---
 sandbox/policy/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 2901c7a1737f7..42cc0518f00a7 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -23,7 +23,7 @@ BASE_FEATURE(kNetworkServiceSandbox,
 // sandbox::MITIGATION_EXTENSION_POINT_DISABLE.
 BASE_FEATURE(kWinSboxDisableExtensionPoints,
              "WinSboxDisableExtensionPoint",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables GPU AppContainer sandbox on Windows.
 BASE_FEATURE(kGpuAppContainer,
-- 
2.39.1.windows.1

