From 1a3ed09a0034e33a68beb84152ef8bcab7313eec Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Wed, 28 Sep 2022 00:00:00 +0000
Subject: [PATCH] disable extension points

---
 sandbox/policy/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox/policy/features.cc b/sandbox/policy/features.cc
index 4ef18a51c00ed..0f13280816eb2 100644
--- a/sandbox/policy/features.cc
+++ b/sandbox/policy/features.cc
@@ -23,7 +23,7 @@ BASE_FEATURE(kNetworkServiceSandbox,
 // sandbox::MITIGATION_EXTENSION_POINT_DISABLE.
 BASE_FEATURE(kWinSboxDisableExtensionPoints,
              "WinSboxDisableExtensionPoint",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enables GPU AppContainer sandbox on Windows.
 BASE_FEATURE(kGpuAppContainer,
-- 
2.39.1.windows.1

