From 63793c3a4d3ce6b74a78515ca6b59ac0d9084230 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sat, 29 Oct 2022 00:00:00 +0000
Subject: [PATCH] Disable keep prefetched suggestions by default

---
 components/ntp_snippets/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/ntp_snippets/features.cc b/components/ntp_snippets/features.cc
index 7bc431d6bcacb..98dedfff29bfa 100644
--- a/components/ntp_snippets/features.cc
+++ b/components/ntp_snippets/features.cc
@@ -72,7 +72,7 @@ const char kNotificationsIgnoredLimitParam[] = "ignored_limit";
 
 BASE_FEATURE(kKeepPrefetchedContentSuggestions,
              "KeepPrefetchedContentSuggestions",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kOptionalImagesEnabledFeature,
              "NTPRemoteSuggestionsOptionalImages",
-- 
2.35.1.windows.2

