From d65f53cd106ab9c1209ca33a8553bdaed2a6f884 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sat, 29 Oct 2022 00:00:00 +0000
Subject: [PATCH] Disable keep prefetched suggestions by default

---
 components/ntp_snippets/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/ntp_snippets/features.cc b/components/ntp_snippets/features.cc
index 0bb6ad032b17d..5770dfe254b32 100644
--- a/components/ntp_snippets/features.cc
+++ b/components/ntp_snippets/features.cc
@@ -68,7 +68,7 @@ const char kNotificationsDailyLimit[] = "daily_limit";
 const char kNotificationsIgnoredLimitParam[] = "ignored_limit";
 
 const base::Feature kKeepPrefetchedContentSuggestions{
-    "KeepPrefetchedContentSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
+    "KeepPrefetchedContentSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kOptionalImagesEnabledFeature{
     "NTPRemoteSuggestionsOptionalImages", base::FEATURE_ENABLED_BY_DEFAULT};
-- 
2.35.1.windows.2

