From 83155a09ff455ebba73ff5dbd47cb4f2edc116af Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sat, 29 Oct 2022 00:00:00 +0000
Subject: [PATCH] Enable disruptive notification permission revoking

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 000eee24411b9..966b8e1d99867 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -352,7 +352,7 @@ BASE_FEATURE(kKeepForceInstalledPreinstalledApps,
 // disruptive notifications.
 BASE_FEATURE(kDisruptiveNotificationPermissionRevocation,
              "DisruptiveNotificationPermissionRevocation",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enable DNS over HTTPS (DoH).
 BASE_FEATURE(kDnsOverHttps,
-- 
2.35.1.windows.2

