From 8045ab9a09b288489e16609df4a69daff7855f5f Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sat, 29 Oct 2022 00:00:00 +0000
Subject: [PATCH] Enable disruptive notification permission revoking

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index f9352a3b9b94a..f2b0a6e9c3cfc 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -346,7 +346,7 @@ BASE_FEATURE(kKeepForceInstalledPreinstalledApps,
 // disruptive notifications.
 BASE_FEATURE(kDisruptiveNotificationPermissionRevocation,
              "DisruptiveNotificationPermissionRevocation",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Enable DNS over HTTPS (DoH).
 BASE_FEATURE(kDnsOverHttps,
-- 
2.35.1.windows.2

