From f5e32f86a92379840bd4d620e8998ba228a54aec Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sat, 29 Oct 2022 00:00:00 +0000
Subject: [PATCH] Enable third party module blocking by default

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index beb42055c61eb..6a5068a45cff5 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -1093,7 +1093,7 @@ BASE_FEATURE(kSupportToolScreenshot,
 // required to start the browser two times to fully enable or disable it.
 BASE_FEATURE(kThirdPartyModulesBlocking,
              "ThirdPartyModulesBlocking",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif
 
 // Disable downloads of unsafe file types over insecure transports if initiated
-- 
2.39.1.windows.1

