From 59df6a3c3fcaeea72f1b1c1bed5522ed66e53c84 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Sun, 8 Jan 2023 00:00:00 +0000
Subject: [PATCH] Enable ThirdPartyStoragePartitioning by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index e89e7e6a011a8..42506df75df4c 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -334,7 +334,7 @@ const base::FeatureParam<bool> kStorageAccessAPIAutoDenyOutsideFPS{
 // by the top level site to reduce fingerprinting.
 BASE_FEATURE(kThirdPartyStoragePartitioning,
              "ThirdPartyStoragePartitioning",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 // Whether to use the new code paths needed to support partitioning Blob URLs.
 // This exists as a kill-switch in case an issue is identified with the Blob
 // URL implementation that causes breakage.
-- 
2.39.1.windows.1

